Software/Development/SemesterProject/App/Include/Constants.h

```diff
@@ -92,8 +92,18 @@
 #define R4_DCLINK_MEAS              (float)10000   // Ohm
 #define R1_CONTROL_SUPPLY_MEAS      (float)8200    // Ohm
 #define R2_CONTROL_SUPPLY_MEAS      (float)1000    // Ohm
-#define TEMP_SENSOR_GAIN            0.01    // 10mV/deg
-
+#define TEMP_SENSOR_GAIN            0.01           // 10mV/deg
+#define MEAS_TO_TEMP                (float)100     // 100deg/V
+
+// Defines of inverse sensor gains
+#define DC_LINK_MEAS_TO_VOLTAGE         (R4_DCLINK_MEAS/R3_DCLINK_MEAS)*((R1_DCLINK_MEAS+R2_DCLINK_MEAS)/R2_DCLINK_MEAS)
+#define CONTROL_SUPPLY_MEAS_TO_VOLTAGE  ((R1_CONTROL_SUPPLY_MEAS+R2_CONTROL_SUPPLY_MEAS)/R2_CONTROL_SUPPLY_MEAS)
+
+// Sensor offsets
+#define CURRENT_SENSOR_OFFSET           9       // [A]

```

Comment:
Shouldnt we make one for each phase? just in case we see there is a considerable difference in the future?